pkg/tikvapi/tikvapi.go

```diff
@@ -36,12 +43,69 @@ const (
 // TiKVClient provides tikv server's api
 type TiKVClient interface {
 	GetLeaderCount() (int, error)
+	FlushLogBackupTasks(ctx context.Context) error
+}
+
+type lazyGRPCConn struct {
+	target string
+	opts   []grpc.DialOption
+}
+
+func (l *lazyGRPCConn) conn(ctx context.Context) (*grpc.ClientConn, error) {
+	ctx, cancel := context.WithTimeout(ctx, DefaultTimeout)
+	defer cancel()
+	ch, err := grpc.DialContext(ctx, l.target, l.opts...)
+	if err != nil {
+		return nil, errors.Annotatef(err, "during connecting to %s", l.target)
+	}
+
+	return ch, nil
 }
 
 // tikvClient is default implementation of TiKVClient
 type tikvClient struct {
-	url        string
-	httpClient *http.Client
+	url           string
+	httpClient    *http.Client
+	grpcConnector *lazyGRPCConn
+}
+
+// FlushLogBackupTasks implements TiKVClient.
+func (c *tikvClient) FlushLogBackupTasks(ctx context.Context) error {
+	logger := klog.FromContext(ctx)
+
+	// For now we are using one-shot sessions, because the `TiKVClient`
+	// interface doesn't provide a `Close` method...
+	conn, err := c.grpcConnector.conn(ctx)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := conn.Close(); err != nil {
+			logger.Error(err, "tikvClient: failed to close grpc connection")
+		}
+	}()
+
+	cli := logbackup.NewLogBackupClient(conn)
+	res, err := cli.FlushNow(ctx, &logbackup.FlushNowRequest{})
+	if err != nil {
+		return err
+	}
```

Comment:
Does all versions of TiKV support this API? what's the behavior if some versions do not support?

Reply:
So, what is the minimal version which supports this?

Reply:
should be 9.0.0, would be picked to 8.5.0 if on demand.